.github/workflows/gin.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
-        go: [1.16, 1.17, 1.18, 1.19]
+        go: ['1.16', '1.17', '1.18', '1.19', '1.20']
```

Comment:
why add `'`?

Reply:
![image](https://user-images.githubusercontent.com/5756338/216237235-372da361-b55f-4ca9-ae64-5a4710bfbaf2.png)

It does not appear to be correctly identified as go1.20 [here](https://github.com/gin-gonic/gin/actions/runs/4071314852/jobs/7013011269).
@thinkerou